_data/ddc_offline_files_2.yaml

```diff
@@ -6,6 +6,14 @@
 - product: "ucp"
   version: "3.0"
   tar-files:
+    - description: "3.0.3 Linux"
+      url: https://packages.docker.com/caas/ucp_images_3.0.3.tar.gz
+    - description: "3.0.3 Windows Server 2016 LTSC"
+      url: https://packages.docker.com/caas/ucp_images_win_2016_3.0.3.tar.gz
+    - description: "3.0.3 Windows Server 1709"
+      url: https://packages.docker.com/caas/ucp_images_win_1709_3.0.3.tar.gz
+    - description: "3.0.3 Windows Server 1803"
+      url: https://packages.docker.com/caas/ucp_images_win_1803_3.0.3.tar.gz
```

Comment:
We also have 3.0.3 IBM Z (s390x) at 
https://packages.docker.com/caas/ucp_images_s390x_3.0.3.tar.gz

Reply:
Resolved, in the Second commit.